### PR TITLE
Feat/select options editable by members

### PR DESCRIPTION
**Pull request by msadura**
https://app.charmverse.io/charmverse/page-3557311048353411

### CODE REVIEW

File: jest.config-server.ts [hunk 1]
```diff
@@ -5,8 +5,8 @@ import { jestConfig } from './jest.config';
 const integrationConfig = {
   ...jestConfig,
   testMatch: [
-    '**/__integration-tests__/**/*.spec.ts',
-    '**/lib/**/*.spec.ts'
+    '**/__integration-tests__/**/*.spec.ts'
+    // '**/lib/**/*.spec.ts'
```
Comment (Devorein) [inline jest.config-server.ts:9]: Yikes we need to uncomment this 👀

File: lib/members/updateMemberProperty.ts [hunk 2]
```diff
@@ -13,6 +13,33 @@ type UpdatePropertyInput = {
...
+  });
+
+  const memberProperty = await prisma.memberProperty.findUnique({
+    where: {
+      id
+    }
+  });
+
+  if (!spaceRole) {
+    throw new InvalidInputError('Invalid workspace data.');
```
Comment (Devorein) [inline lib/members/updateMemberProperty.ts:30]: If the user is not a member of the workspace its better to throw `UserIsNotSpaceMemberError`
image

**Review by Devorein — CHANGES_REQUESTED** [review]

File: jest.config-server.ts (hunk 1 above)
Comment (msadura) [reply jest.config-server.ts:9]: actually it was commented all the time, I just uncommented this in my previous pr by mistake

**Review by Devorein — APPROVED** [review]

File: jest.config-server.ts (hunk 1 above)
Comment (mattcasey) [reply jest.config-server.ts:9]: I think I asked this once before, can we just remove this line?

Comment (msadura) [reply jest.config-server.ts:9]: We can probably remove it, but I do not see other way to run `libs/...` tests via vscode jest runner extension. Maybe I am just missing some piece of the puzzle.

**Pull request merged**